Fix up journalctl examples (#2691)
sensu/sensu-docs: content/sensu-go/5.19/operations/maintain-sensu/troubleshoot.md

Use the `journald` keyword `since` to refine the basic `journalctl` commands and narrow your search by timeframe.

Retrieve all the logs for sensu-backend since yesterday:

{{< code shell >}}
journalctl -u sensu-backend --since yesterday | tee sensu-backend-$(date +%Y-%m-%d).log
{{< /code >}}

Retrieve all the logs for sensu-agent since a specific time:

{{< code shell >}}
journalctl -u sensu-agent --since 09:00 --until "1 hour ago" | tee sensu-agent-$(date +%Y-%m-%d).log
{{< /code >}}

Retrieve all the logs for sensu-backend for a specific date range:

{{< code shell >}}
journalctl -u sensu-backend --since "2015-01-10" --until "2015-01-11 03:00" | tee sensu-backend-$(date +%Y-%m-%d).log
{{< /code >}}

{{< platformBlockClose >}}

{{< platformBlock "Windows" >}}

##### Logging edge cases

> In some cases where a Sensu service experiences a panic crash, the service may seem to start and stop without producing any output in journalctl. This is due to a [bug in systemd][12].

Reviewer: ```suggestion
If a Sensu service experiences a panic crash, the service may seem to start and stop without producing any output in journalctl.
This is due to a [bug in systemd][12].
```

> In these cases, you can try using the _COMM variable instead of the -u flag, to access additional log entries:

Reviewer: ```suggestion
In these cases, try using the `_COMM` variable instead of the `-u` flag to access additional log entries:
```
